e2e/util/namespace.go

```diff
@@ -17,7 +17,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func CreateNamespace(client client.Client, namespace string) error {
+const volsyncPrivilagedMovers = "volsync.backube/privileged-movers"
```

Comment:
Lets add a comment explaining this annotation with a link to https://volsync.readthedocs.io/en/stable/usage/permissionmodel.html#controlling-mover-permissions

Reply:
added